server/rosetta/client_online.go

```diff
@@ -468,3 +477,16 @@ func (c *Client) blockTxs(ctx context.Context, height *int64) (crgtypes.BlockTra
 		Transactions:  finalTxs,
 	}, nil
 }
+
+func (c *Client) getRealHeight(ctx context.Context, height *int64) (realHeight *int64, err error) {
```

Comment:
nit: As opposed to fake height? Maybe call this `getInitialHeight` or just `getHeight`?

Reply:
^ agreed with @alexanderbez, I'd also comment the function even if it's private, so future mantainers can understand what the purpose of this function is.

Reply:
i just renamed it :)